subcommand/delete-completed-job/command.go

```diff
@@ -95,10 +94,7 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
-	logger := hclog.New(&hclog.LoggerOptions{
-		Level:  hclog.Info,
-		Output: os.Stderr,
-	})
+	logger := hclog.Default()
```

Comment:
This is equivalent.

Reply:
Should we use the constructor here?

Reply:
You mean swap it back to hclog.New() or use `common.Logger("info")?

Reply:
use `common.Logger(hclog.Info)` for consistency?

Reply:
Not a huge deal for me one way or the other btw.

Reply:
Good idea because we might add more to that function and expect it to apply to all loggers.